### ISSUE TITLE
no_drawer: true in _config.yml pins the drawer above the rest of the content

### ISSUE THREAD

**Issue by inspectordanno** [qwtel/hydejack#163]
I think I might have encountered a bug with the drawer in _config.yml. My goal is to have the default behavior of the drawer be open. I.e., the drawer should be on the left side of the screen, and the content should be on the right side of the screen.

In the past, I think I did this in _config.yml by enabling` no_drawer: true`. But when I tried it this time, I got this weird result:

<img width="1440" alt="Screen Shot 2019-05-01 at 9 08 46 PM" src="https://user-images.githubusercontent.com/13577558/57052604-650f4e80-6c56-11e9-89f2-86f7ce19792a.png">

Basically, the drawer ceased to be a slider, and just pinned itself above the rest of the content. In addition, there is a gray "sidebar" where the drawer should be.

I thought it could just be a problem with my files, but I downloaded the starter kit and it is exhibiting the same behavior.

Is this an intended feature? Is there a way to have the default behavior of the drawer be open like before?

**Comment by inspectordanno** [comment]
I figured out the issue! In the welcome layout, set

`cover: false`

This will change the default drawer behavior to open.
